tcp-to-blob/src/tcp-to-blob.ts

```diff
@@ -59,10 +59,11 @@ if (require.main === module) {
             ) {
                 sender = 'canary'
             }
-            const filename = `tcp_data_${timestampStr}_${sender}_${remoteToken}.${sender=='canary'? 'txt':'bin'}`
+            const localFileName = `tcp_data_${timestampStr}_${sender}_${remoteToken}.${sender=='canary'? 'txt':'bin'}`
+            const filename = `${sender}/${localFileName}`
             const logger = makeLogger({
                 subsystem: 'tcp-to-blob',
-                filename,
+                localFileName,
```

Comment:
This should still use `filename` to prevent breaking dashboards or other kusto queries.

Reply:
Fixed. Thanks for the input